mentat/llm_api.py

```diff
@@ -24,12 +25,10 @@ def setup_api_key():
         openai.api_key = key
         openai.Model.list()  # Test the API key
     except openai.error.AuthenticationError:
-        cprint(
+        raise UserError(
```

Comment:
We call this function before the try except in app.py; we should move that call into loop() so this error gets properly caught

Reply:
Good catch